src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodCustomMessage.java

```diff
@@ -0,0 +1,33 @@
+/*
+JavadocMethod
+allowedAnnotations = (default)Override
+validateThrows = (default)false
+accessModifiers = (default)public, protected, package, private
+allowMissingParamTags = (default)false
+allowMissingReturnTag = (default)false
+tokens = (default)METHOD_DEF, CTOR_DEF, ANNOTATION_FIELD_DEF, COMPACT_CTOR_DEF
+message.javadoc.return.expected = @return tag should be present and have description :)
+message.javadoc.expectedTag = Expected {0} tag for ''{1}'' :)
+message.javadoc.unusedTag = Unused {0} tag for ''{1}'' :)
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod;
+
+public class InputJavadocMethodCustomMessage {
+
+    /** missing return **/
+    int method3() { // violation
```

Comment:
Please add BDD style comments in this file, we should be doing this for all inputs now to be consistent.

Reply:
// Violation 'the violation message' ?

Reply:
Done.